equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/report/Crm4PeriodicalReportController.java

```diff
@@ -25,26 +20,12 @@ public class Crm4PeriodicalReportController implements ReportCrm4Api {
     @Override
     public ResponseEntity<String> generateReportCrm4(
             String decisionFrom, String decisionTo, String profileAcceptedTypes) {
-        checkRequestInputs(decisionFrom, decisionTo, profileAcceptedTypes);
+        ReportInputsUtil.checkInputs(decisionFrom, decisionTo,
+                profileAcceptedTypes, String.valueOf(CRM_TYPE_4)
```

Comment:
`String.valueOf(CRM_TYPE_4)` - use `static import` where required.

Reply:
I don't understand.  But I don't thing it is a good idea to use static import to a function call.